pkg/repository/udmrepo/repo_options.go

```diff
@@ -88,6 +90,8 @@ type RepoOptions struct {
 	GeneralOptions map[string]string
 	// StorageOptions takes storage specific options
 	StorageOptions map[string]string
+	// EagerGC will overwrite kopia maintenance interval to 12/6 hours
+	EagerGC bool
```

Comment:
need to research how/where to set this option from the configmap

Reply:
Yes, better a more informative value instead of `bool`, for extensibility consideration.